.circleci/config.yml

```diff
@@ -19,6 +20,12 @@ jobs:
           sleep 5
           docker port vault
           docker logs vault
+    - run:
+        name: "Start MySQL Container"
+        command: |
+          docker run --detach -e MYSQL_ROOT_PASSWORD=mysql --name mysql -p 3306:3306 mysql:latest
+          sleep 30
```

Comment:
Instead of sleeping for exact 30 seconds you could use [dockerize](https://github.com/jwilder/dockerize):
```sh
dockerize -wait tcp://127.0.0.1:3306 -timeout 30s
```